meetings/working-groups/extensions/content-based-naming.md

```diff
@@ -0,0 +1,217 @@
+## Summary
+
+The current metadata strategy for skeleton assemblies relies on ordinals to group skeleton types generated from extension blocks. The use of ordinals causes friction in a few parts of the C# ecosystem: 
+
+- Edit and Continue: any use of ordinals tied to source ordering means that adding or removing elements can create unnecessary, and possibly unresolvable, conflicts during ENC operations. Natural operations like adding a new extension block in the middle of two others can lead to renaming of existing blocks which is difficult to reconcile.
+- Public API Tracking: the skeleton assemblies are necessarily generated as `public` APIs and there are many tools in the .NET ecosystem that track `public` API usage. The current design means moving types around in source can observably change the set of `public` API in an assembly. Shipping with this design would mean this ecosystem of tools would need to change to treat this `public` API as _special_ in some way.
```

Comment:
Again a reference to a "skeleton assemblies", I assume you meant to refer to nested types representing extension blocks.